/**:
  ros__parameters:
    camera_namespaces: ["camera6", "camera7"]
    message_lifespan: 0.09
    approximate_sync: false
